### ISSUE TITLE
Clark - Updater problem

### ISSUE THREAD

**Issue by espinoj** [cmu-phil/tetrad#579]
Updater Problem

The Updater connected to the contents of the Estimation box, does nothing for SEM models. See attached Updater Problem.   It should estimate the new means of variables it effects. Note in the example, the mean of the effect, X2 does not change.

**Comment by jdramsey** [comment]
@yuanzhou @cg09  No, this is working in the development branch. (Sorry, I realized this is an algorithmic issue.) See picture below. I made a SEM IM, added an updater, then set the updated value of X7 to 8.0, and you see, the mean of X7 is set to 8.0. I don't know what was happening before, but it's working now.

<img width="976" alt="Screen Shot 2019-04-10 at 6 53 04 AM" src="https://user-images.githubusercontent.com/9853255/55873314-5228c300-5b5d-11e9-927f-7830fc7c95bc.png">

I think this can be closed, unless someone else can break it.
